### PR TITLE
Improve Install Instructions

### PR DESCRIPTION
**Pull request by aditirajagopal**
Resolves: #19235
Signed-off-by: Aditi Rajagopal arajagopal@us.ibm.com

### CODE REVIEW

File: docs/installation/linux/debian.md [hunk 1]
```diff
@@ -21,6 +21,12 @@ Docker is supported on the following versions of Debian:
  >**Note**: If you previously installed Docker using `APT`, make sure you update
  your `APT` sources to the new `APT` repository.
 
+This page instructs you to install using Docker-managed release packages and
+installation mechanisms. Using these packages ensures you get the latest release
+of Docker. If you wish to install using Debian-managed packages, consult your
+Debian documentation. If you are comfortable with package managers, and are familiar
+with Docker's prerequisites, feel free to skip to step 3 in the [Install](https://docs.docker.com/engine/installation/debian/#install-docker) section
```
Comment (tophj-ibm) [inline docs/installation/linux/debian.md]: The # url id selector doesn't work for me on any of the links except the centos one, looks like the element id has some sort of hash value appended to it.

Comment (aditirajagopal) [inline docs/installation/linux/debian.md]: Yeah they recently changed the file structure with the installation docs...made the anchor tags all janky: https://docs.docker.com/engine/installation/linux/rhel/#install-with-the-script:d0deea4e0f79eb0964a0cfafb261e5f3

**Comment by MHBauer** [comment]
I think the hash links can just be only the hash link. It should link to the same page when it's rendered. That way the links are internal, and not hardcoded.

**Comment by MHBauer** [comment]
links checked in `make docs` and LGTM.

**Comment by MHBauer** [comment]
@moxiegirl @thaJeztah for docs

File: docs/installation/linux/centos.md [hunk 2]
```diff
@@ -19,7 +19,8 @@ or support Docker on these distributions.
 This page instructs you to install using Docker-managed release packages and
 installation mechanisms. Using these packages ensures you get the latest release
 of Docker. If you wish to install using CentOS-managed packages, consult your
-CentOS documentation.
+CentOS documentation. If you are comfortable with package managers, and are
```
Comment (thaJeztah) [inline docs/installation/linux/centos.md]: Hm looks like this describes the opposite as what the link refers to; the `curl | sh` is more meant for people that are *not* comfortable with package managers, and not familiar with the prerequisites.

File: docs/installation/linux/debian.md [hunk 3]
```diff
@@ -21,6 +21,12 @@ Docker is supported on the following versions of Debian:
  >**Note**: If you previously installed Docker using `APT`, make sure you update
  your `APT` sources to the new `APT` repository.
 
+This page instructs you to install using Docker-managed release packages and
+installation mechanisms. Using these packages ensures you get the latest release
+of Docker. If you wish to install using Debian-managed packages, consult your
+Debian documentation. If you are comfortable with package managers, and are familiar
+with Docker's prerequisites, feel free to skip to step 3 in the [Install](#install-docker) section
```
Comment (thaJeztah) [inline docs/installation/linux/debian.md]: I think the original issue was that we don't mention the `curl | sh` method on the Ubuntu and Debian pages; only skipping the prerequisites section doesn't solve that.

If we want to be consistent, I think we should

- Add a section for the `curl | sh` "quick install" method (possibly mentioning its limitations)
- Add a link from that section to the "manual installation" method for those comfortable with package managers to do the manual steps

Also see my comment on the linked issue https://github.com/docker/docker/issues/19235#issuecomment-171010218

File: docs/installation/linux/rhel.md [hunk 4]
```diff
@@ -16,7 +16,9 @@ Docker is supported on Red Hat Enterprise Linux 7. This page instructs you to
 install using Docker-managed release packages and installation mechanisms. Using
 these packages ensures you get the latest release of Docker. If you wish to
 install using Red Hat-managed packages, consult your Red Hat release
-documentation for information on Red Hat's Docker support.
+documentation for information on Red Hat's Docker support. If you are comfortable
+with package managers, and are familiar with Docker's prerequisites, feel free
+to skip to step 3 in the [Install with the script](#install-with-the-script) section.
```
Comment (thaJeztah) [inline docs/installation/linux/rhel.md]: Same comment as the CentOS page https://github.com/docker/docker/pull/20046/files#r52120837

File: docs/installation/linux/ubuntulinux.md [hunk 5]
```diff
@@ -21,7 +21,8 @@ Docker is supported on these Ubuntu operating systems:
 This page instructs you to install using Docker-managed release packages and
 installation mechanisms. Using these packages ensures you get the latest release
 of Docker. If you wish to install using Ubuntu-managed packages, consult your
-Ubuntu documentation.
+Ubuntu documentation. If you are comfortable with package managers, and are familiar
+with Docker's prerequisites, feel free to skip to step 3 in the [Install](#install) section
```
Comment (thaJeztah) [inline docs/installation/linux/ubuntulinux.md]: Same comment here as the Debian page https://github.com/docker/docker/pull/20046/files#r52120886

**Comment by thaJeztah** [comment]
closing this because this is currently not right, and the PR hasn't been updated in a while

**Pull request closed without merging**